FiniteCocompletions/gap/CategoryOfColimitQuivers.gi

```diff
@@ -59,12 +59,30 @@ InstallMethod( CategoryOfColimitQuivers,
     ##
     object_datum := { ColimitQuivers, o } -> DefiningPairOfColimitQuiver( o );
     
+    ##
+    object_datum_type :=
+      CapJitDataTypeOfNTupleOf( 2,
+              CapJitDataTypeOfListOf( CapJitDataTypeOfObjectOfCategory( C ) ),
+              CapJitDataTypeOfListOf(
+                      CapJitDataTypeOfNTupleOf( 3,
+                              rec( filter := IsInt ),
+                              CapJitDataTypeOfMorphismOfCategory( C ),
+                              rec( filter := IsInt ) ) ) );
+    
```

Comment:
In general I would put this above `object_constructor` because it is the type of the input of the object constructor and thus semantically comes first.

Reply:
Good idea, done.